ts/Core/Series/Series.ts

```diff
@@ -4104,14 +4112,20 @@ class Series {
             typeof options.pointStart !== 'undefined' ||
             typeof options.pointInterval !== 'undefined' ||
             typeof options.relativeXValue !== 'undefined' ||
+
+            // Check if below `plotOptions.series` props are defined (#19203)
+            typeof plotOptions.series?.dataGrouping !== 'undefined' ||
+            typeof plotOptions.series?.pointStart !== 'undefined' ||
+            typeof plotOptions.series?.pointInterval !== 'undefined' ||
+            typeof plotOptions.series?.pointIntervalUnit !== 'undefined' ||
+            typeof plotOptions.series?.keys !== 'undefined' ||
+
```

Comment:
I think those lines are not necessary for that fix. It is checked in `hasOptionsChanged` method (`defined`) and also currently (in these lines) you are not checking plot options for exact series type (e.g. column). After removing these lines all tests pass.